src/Routes/Routes.tsx

```diff
@@ -0,0 +1,20 @@
+import { createBrowserRouter } from "react-router-dom";
+import App from "../App";
+import HomePage from "../Pages/HomePage/HomePage";
+import CasePage from "../Pages/CasePage/CasePage";
+import PricePage from "../Pages/PricePage/PricePage";
+import AboutPage from "../Pages/AboutPage/AboutPage";
+import BlogPage from "../Pages/BlogPage/BlogPage";
+
+
+export const router = createBrowserRouter([{
+    path:'/',
+    element: <App/>,
+    children:[
+        {path:'/', element:<HomePage/>},
```

Comment:
and here you could reuse the paths constant

Reply:
Used new paths' file for making routing.